flixel/input/gamepad/FlxGamepadInputID.hx

```diff
@@ -81,14 +81,24 @@ abstract FlxGamepadInputID(Int) from Int to Int
 	/**for a mouse-like input such as touch or IR camera. Vertical axis.**/
 	var POINTER_Y            = 29;
 	
+	var LEFT_STICK_DIGITAL_UP          = 30; 
+	var LEFT_STICK_DIGITAL_RIGHT       = 31;
+	var LEFT_STICK_DIGITAL_DOWN        = 32;
+	var LEFT_STICK_DIGITAL_LEFT        = 33;
+	
+	var RIGHT_STICK_DIGITAL_UP         = 34; 
+	var RIGHT_STICK_DIGITAL_RIGHT      = 35;
+	var RIGHT_STICK_DIGITAL_DOWN       = 36;
+	var RIGHT_STICK_DIGITAL_LEFT       = 37;
+	
 	/**an extra digital button that doesn't fit cleanly into the universal template**/
-	var EXTRA_0              = 30;
+	var EXTRA_0              = 46;
```

Comment:
Changing these IDs is a breaking change because this abstract allows `from Int` and `to Int`.